src/Domain/Creators/TimeDependence/TimeDependence.hpp

```diff
@@ -56,7 +61,13 @@ struct TimeDependence {
  private:
   using creatable_classes_1d = tmpl::list<>;
   using creatable_classes_2d = tmpl::list<UniformRotationAboutZAxis<2>>;
-  using creatable_classes_3d = tmpl::list<UniformRotationAboutZAxis<3>>;
+  using creatable_classes_3d = tmpl::list<
+      UniformRotationAboutZAxis<3>,
+      Composition<
+          TimeDependenceCompositionTag<CubicScale<3>,
+                                       std::numeric_limits<size_t>::max()>,
+          TimeDependenceCompositionTag<UniformRotationAboutZAxis<3>,
+                                       std::numeric_limits<size_t>::max()>>>;
```

Comment:
(not a change request)
Does C++ allow for some way of omitting the `Suffix` template parameter here so it defaults to `std::numeric_limits<size_t>::max()` which is set in `Composition.hpp`, but still using the forward-declaration instead of including the header?
That said, you're including `Composition.hpp` at the bottom of this file anyway, so you might as well move the include up and delete the default `Suffix`.